packages/react-storage/src/components/StorageBrowser/composables/DataTable/DataTable.tsx

```diff
@@ -49,44 +51,46 @@ export const DataTable = ({
     }
   });
 
-  const mappedRows = rows.map(({ key, content }) => ({
-    key,
-    content: content.map(({ key, content, type }) => {
-      switch (type) {
-        case 'button': {
-          return {
-            key,
-            content: <ButtonDataCell content={content} />,
-          };
-        }
-        case 'checkbox': {
-          return {
-            key,
-            content: <CheckboxDataCell content={content} />,
-          };
-        }
-        case 'date': {
-          return {
-            key,
-            content: <DateDataCell content={content} />,
-          };
-        }
-        case 'number': {
-          return {
-            key,
-            content: <NumberDataCell content={content} />,
-          };
-        }
-        case 'text':
-        default: {
-          return {
-            key,
-            content: <TextDataCell content={content} />,
-          };
-        }
-      }
-    }),
-  }));
```

Comment:
Only change is the addition of the ternary, returning an empty array instead when loading. Rows content is not changed